1904-11-03.xml

```diff
@@ -0,0 +1,2652 @@
+                    themselves at the station at 11.30 a.m. next Tuesday to greet hisHighness on his
+                    arrival. The Ministry of War has given orders for a detachment to be' present as
+                    a guard of () The train in which his Highness travels will stop at Damanhour,
+                    Tantah, (Buket el) San, and Benha. </p>
+            </div>
+            <div type="advert">
+                <head>WINDSOR HOTEL. </head>
+                <p>Largest 1 class and most comfortable hotel in Alexandria. Facing the sea Central
+                    position. Under English management. 81.11.94 </p>
+            </div>
+            <div type="advert">
+                <head>BECK &amp;amp CO'S PILSENER BEER </head>
+                <p>BREMEN. </p>
+                <p>FEARS NO HONEST COMPETITION FOR QUALITY. </p>
+                <p>N.B.—Inferior Brande now being offered to Manager of certain good cercles. </p>
+                <p>Beware of evilly disposed competitors running down this very Superior Brand of
+                    Beer. </p>
+            </div>
+            <div type="advert">
+                <head>EDEN PALACE HOTEL, </head>
+                <p>CAIRO. </p>
+                <p>Open all the year round. </p>
+                <p>ELECTRIC LIGHT, —LIFT, —MODERATE CHARGES</p>
+            </div>
+            <div type="section">
+                <head> LOCAL AND GENERAL. </head>
+
+                <div type="item">
+                    <head/>
+                    <p>RAMADAN begins on Tuesday next.</p>
+                    <div type="item">
+                        <head/>
+                        <p>THE SLEENAN BASE is to be heard at Alexandria by Judge Saumarez on the
+                            10th inst.</p>
+                    </div>
+                    <div type="item">
+                        <head>LEGISLATIVE COUNCIL.-</head>
+                        <p>The next meeting of the Legislative Council is not expected to take place
+                            before December 1.</p>
+                    </div>
+                    <div type="item">
+                        <head>AMERICAN MISSION.- </head>
+                        <p>The English services in Tantah under the auspices of the American Mission
+                            will be resumed next Sabbath, 6th inst., at 11 a.m., and will be held as
+                            formerly once every two weeks. </p>
+                    </div>
+                    <div type="item">
+                        <head>SHOOTING OUTRAGE.</head>
+                        <p>The director of the sugar factory at Bibeh has been fired at and slightly
+                            wounded while standing by his window in the evening. A dissatisfied or
+                            dismissed workman is believed to have been guilty of the crime.</p>
+                        <div type="item">
```

Comment:
same issue as previous comment. You are starting this item *inside* the previous item. You need to close the previous item with a </div>